example/console-example/WinToast Console Example.vcxproj

```diff
@@ -17,18 +17,26 @@
       <Configuration>Release</Configuration>
       <Platform>x64</Platform>
     </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|ARM64">
+      <Configuration>Debug</Configuration>
+      <Platform>ARM64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|ARM64">
+      <Configuration>Release</Configuration>
+      <Platform>ARM64</Platform>
+    </ProjectConfiguration>
   </ItemGroup>
   <PropertyGroup Label="Globals">
     <VCProjectVersion>15.0</VCProjectVersion>
     <ProjectGuid>{7C5AC60D-8668-47B6-9D05-FB363F854065}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
-    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
```

Comment:
Will this change anything with regard to Windows 8 support?

If so, would it make sense to override that platform in a `<PropertyGroup Include="*|ARM64">`-like section instead?

Reply:
`WindowsTargetPlatformVersion`, despite its name, specifies the version of the SDK to use. 10.x SDK does support Windows 8.1: https://developer.microsoft.com/en-us/windows/downloads/windows-sdk/

Reply:
Seeing as the unmodified binary produces the same result, I'd say the Icon is probably unrelated to this PR.

Reply:
@dennisameling thank you for digging deeper. I agree that it is unrelated, and if you find the time to open that follow-up PR in build-extra, I will be delighted to merge it!